metadata-ingestion/src/datahub/api/entities/corpgroup/corpgroup.py

```diff
@@ -65,10 +67,12 @@ class CorpGroup(BaseModel):
     overrideEditable: bool = False
     picture_link: Optional[str] = None
     slack: Optional[str] = None
-    admins: List[Union[str, CorpUser]] = []
```

Comment:
lets not make this a breaking change- just remove it from docs and mark it as deprecated please.

Reply:
Already taken care of by this line
```
_rename_admins_to_owners = pydantic_renamed_field("admins", "owners")
```

Reply:
nice!